src/Symfony/Component/ExpressionLanguage/ParserCache/ArrayParserCache.php

```diff
@@ -15,6 +15,8 @@
 
 /**
  * @author Adrien Brault <adrien.brault@gmail.com>
+ *
+ * @deprecated ParserCacheInterface is deprecated. Use Symfony\Component\Cache\Adapter\ArrayAdapter instead
```

Comment:
you should keep `@deprecated`! just don't add this `@trigger_error(..., E_USER_DEPRECATED)` in the ParserCacheInterface.php file.